outputs/core_pools.json

```diff
@@ -67,7 +67,6 @@
   },
   "zkevm": {
     "0x1d0a8a31cdb04efac3153237526fb15cc65a252000000000000000000000000f": "B-rETH-STABLE",
-    "0x65da876a95cd5b6a5880710628c436409b1b29fa00000000000000000000005b": "wstETH/ankrETH ",
```

Comment:
@Xeonus this looks like a BPT pricing issue?  I imagine it is being removed here because it doesn't meet AUM requirements.